src/otx/algorithms/detection/configs/detection/cspdarknet_yolox_l/hpo_config.yaml

```
@@ -0,0 +1,16 @@
+metric: mAP
+search_algorithm: asha
+early_stop: None
+hp_space:
+  learning_parameters.learning_rate:
+    param_type: qloguniform
+    range:
+      - 0.00002
+      - 0.002
+      - 0.00001
```

```suggestion
      - 0.0001
      - 0.01
      - 0.0001
```
I suggest you to change values as above.